src/bridge/cockpitpackages.c

```diff
@@ -374,12 +374,30 @@ expand_libexec (const gchar *variable,
   return NULL;
 }
 
+static void
+apply_override (JsonObject *manifest,
+                const char *directory,
+                const char *filename)
+{
+  g_autoptr(GError) error = NULL;
+
+  g_autoptr(JsonObject) override = read_json_file (directory, filename, &error);
+  if (error)
```

Comment:
I know you're just copying this, but the normal paradigm is to check the return value.

Reply:
Good point, fixed. As repentance I cleaned up read_json_file() some more.